### PR TITLE
Update Connection.class.php

### PR DESCRIPTION
**Pull request by ComputerScienceMasterStudent**
use mysql_escape_string to prevent SQL injection

### CODE REVIEW

File: Connection.class.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function close() {
      */
     public function query($sql) {
         $this->connect();
-        $result = parent::query($sql);
+        $result = parent::query(mysql_escape_string($sql));
```
Comment (lucasrenan) [inline Connection.class.php]: hey @studentmsc2018,
first of all, thanks for your contribution.

according to [PHP docs](http://php.net/manual/en/function.mysql-escape-string.php), `mysql_escape_string` is deprecated.  you should use `mysqli::escape_string`. for this case `self::escape_string` should work fine.

could you please apply the changes?

**Review by lucasrenan — CHANGES_REQUESTED** [review]

**Comment by ComputerScienceMasterStudent** [comment]
you are right. I prepared a new commit commit 0dfe31394e357e319bed363c9d24ab4fa2ae1b32

File: Connection.class.php (hunk 1 above)
Comment (ComputerScienceMasterStudent) [reply Connection.class.php]: you are right. I prepared a new commit commit 0dfe313

Comment (ComputerScienceMasterStudent) [reply Connection.class.php]: you are right. I prepared a new commit commit 0dfe313

**Review by lucasrenan — APPROVED** [review]
@studentmsc2018 thanks for your contribution :)

**Pull request merged**